.npmrc

```diff
@@ -1,2 +1,3 @@
 engine-strict = true
+package-lock = false
```

Comment:
Goodbye `npm-shrinkwrap.json` and `package-lock.json`. 👋

Reply:
I don't think you need that file anymore, you can have a file `.yarnrc` also, one 1 option you might be interested:
`--install.check-files true`
install will verify file tree of packages for consistency

Reply:
See my other comment: https://github.com/embark-framework/embark/pull/1098#discussion_r234409051